core/src/main/java/org/apache/iceberg/util/SortOrderUtil.java

```diff
@@ -74,6 +101,28 @@ public static SortOrder buildSortOrder(Schema schema, PartitionSpec spec, SortOr
     return builder.build();
   }
 
+  private static Map<Pair<Transform<?, ?>, Integer>, PartitionField> requiredClusteringFields(PartitionSpec spec) {
+    Map<Pair<Transform<?, ?>, Integer>, PartitionField> requiredClusteringFields = Maps.newLinkedHashMap();
+    for (PartitionField partField : spec.fields()) {
+      if (!partField.transform().toString().equals("void")) {
```

Comment:
What's our preferred way to check if its a void transform? Here is what we have in `Partitioning`:

```
field.transform().equals(Transforms.alwaysNull())
```

Reply:
That's probably better. I'll update to that.